### DISCUSSION TITLE
Rom Hacks

### DISCUSSION THREAD

**Discussion by DarkNM** [OnionUI/Onion#798] [Q&A]
Are we able to run rom hacks if we add them? Im really looking forward to playing Pokémon Unbound on the Miyoo Mini Plus.

**Comment by murmelbahn** [comment]
I tried a Super Mario World rom hack and it works fine. In the end its just like a regular game. Give it a try.

**Comment by dreams897** [comment]
@DarkNM yeah absolutely, if you play Unbound you should definitely set the clock first then manually launch it through Retroarch (mGBA core) EACH time, never do a regular launch through the game list as it will mess things up. Also don't set the in-game audio quality past medium and don't use frame-skip
